### PR TITLE
index: switch LibreSSL part for libc.

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

**Comment by Vaelatern** [comment]
LGTM.

File: index.html [hunk 1]
```diff
@@ -47,12 +47,11 @@ <h3>runit</h3>
...
-        <div class="row">
-                <div class="col-md-4">
-                        <h3>LibreSSL</h3>
-                        <p>We were the first distribution to switch to <em>LibreSSL</em> by default, replacing <em>OpenSSL</em>.</p>
-			<p>Due to the <a href="https://en.wikipedia.org/wiki/Heartbleed">Heartbleed</a> fiasco, we believe that the <a href="https://www.openbsd.org">OpenBSD</a> project has qualified and <em>pro-active</em> developers to provide a more secure alternative.</p>
-		</div>
+		<div class="row">
+			<div class="col-md-4">
+				<h3>Musl and glibc</h3>
+				<p>Void Linux supports both the <a href="http://musl.libc.org/">Musl</a> and <a href="https://www.gnu.org/software/libc/">glibc</a> libcs, patching incompatible projects when necessary and working with upstream projects to make their code bases more standards compliant and compatible.
```
Comment (ahesford) [inline index.html:53]: "libcs" is a strange construct. I recommend "C libraries".

Comment (ericonr) [reply index.html:53]: Perhaps "C standard libraries"?

Comment (ericonr) [reply index.html]: Should be fixed now.

Comment (ahesford) [reply index.html]: Looks good to me.

**Comment by ericonr** [comment]
Ping

**Pull request merged**